robot_smach_states/examples/test-detect-faces

```diff
@@ -7,22 +7,16 @@ import sys
 import rospy
 
 # Detect face
+from robot_skills import get_robot
 from robot_smach_states import DetectFace
 
 
 if __name__ == "__main__":
 
     rospy.init_node('test_detect_faces')
     robot_name = sys.argv[1]
```

Comment:
Add argument check like other files.

Reply:
See a71172c306547eba6d50c057baaef8c0ec10a1d0 and 106535870505f785620744139c45820da4f330a4